src/Database/PostgreSQL/ORM/Validations.hs

```diff
@@ -9,16 +9,19 @@ import Data.Monoid
 #endif
 import qualified Data.Text as T
 import Data.Typeable
+import Data.Semigroup
```

Comment:
Just add a version gate for this so it works on older GHC versions as well.

Reply:
I hope now it is better